src/components/layouts/DashboardLayout/components/DashboardHeader/styles.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'aphrodite/no-important'
+import { colors } from '../../../../../theme/constants'
+
+const styles = StyleSheet.create({
+  container: {
+    height: 175,
+    backgroundColor: colors.darkBg,
+    color: colors.white
+  },
+  navbar: {
+    height: 36
+  },
+  notifications: {
+    marginLeft: 'auto'
+  },
+  separator: {
+    width: 1,
+    height: '75%',
+    backgroundColor: colors.gray
+  },
+  deviceInfo: {
+    alignItems: 'flex-end'
+  },
+  deviceName: {
```

Comment:
are not these values pre-defined in theme? if they are, it's better to use `tx` styles, if they are not, can't we add em?